airbyte-cdk/python/airbyte_cdk/entrypoint.py

```diff
@@ -106,6 +106,9 @@ def run(self, parsed_args: argparse.Namespace) -> Iterable[str]:
                     raw_config = self.source.read_config(parsed_args.config)
                     config = self.source.configure(raw_config, temp_dir)
 
+                    yield from [
```

Comment:
Emitting right after `source.configure` is it's often there the config migration occurs. See example of migration [here](https://github.com/airbytehq/airbyte/pull/33104)

Reply:
~are we currently not emitting those?~
this makes sense. source-s3 is just printing the message now.

Reply:
Just to be clear, we would emit them before emitting the command output or the first read result or if an exception is raised. However, I was afraid of the interpreter completely breaking and skipping error handling. One example of this is:

```
>>> import ast
>>> try:
...     ast.literal_eval('+0'*10**6)
... except Exception:
...     print("Exception caught")
...
zsh: segmentation fault  python
```